src/System.Linq.Queryable/src/project.json

```diff
@@ -1,23 +1,19 @@
 {
-  "frameworks": {
-    "netstandard1.3": {
-      "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.0.1",
-        "System.Collections": "4.0.11",
-        "System.Diagnostics.Debug": "4.0.11",
-        "System.Linq": "4.1.0",
-        "System.Linq.Expressions": "4.1.0",
-        "System.Reflection": "4.1.0",
-        "System.Reflection.Extensions": "4.0.1",
-        "System.Resources.ResourceManager": "4.0.1",
-        "System.Runtime": "4.1.0",
-        "System.Threading.Tasks": "4.0.11"
-      }
+    "dependencies": {
+      "Microsoft.NETCore.Platforms": "1.2.0-beta-24711-02",
+      "System.Collections": "4.4.0-beta-24711-02",
+      "System.Diagnostics.Debug": "4.4.0-beta-24711-02",
+      "System.IO": "4.4.0-beta-24711-02",
+      "System.Linq": "4.4.0-beta-24711-02",
+      "System.Linq.Expressions": "4.4.0-beta-24711-02",
+      "System.Reflection": "4.4.0-beta-24711-02",
+      "System.Reflection.Extensions": "4.4.0-beta-24711-02",
+      "System.Resources.ResourceManager": "4.4.0-beta-24711-02",
+      "System.Runtime": "4.4.0-beta-24711-02",
+      "System.Threading.Tasks": "4.4.0-beta-24711-02"
```

Comment:
What's the guidance for listing dependencies? I'm not convinced this project has a dependency on `System.Threading.Tasks` for example. Do we want to keep these at a minimum?